src/redux/tablesRedux.js

```diff
@@ -0,0 +1,105 @@
+import Axios from 'axios';
+import { api } from '../settings.js';
+
+/* selectors */
+export const getAll = ({tables}) => tables.data;
+export const getLoadingState = ({tables}) => tables.loading;
+
+/* action name creator */
+const reducerName = 'tables';
+const createActionName = name => `app/${reducerName}/${name}`;
+
+/* action types */
+const FETCH_START = createActionName('FETCH_START');
+const FETCH_SUCCESS = createActionName('FETCH_SUCCESS');
+const FETCH_ERROR = createActionName('FETCH_ERROR');
+const UPDATE_STATUS = createActionName('UPDATE_STATUS');
+const UPDATE_ERROR = createActionName('UPDATE_ERROR');
+
+/* action creators */
+export const fetchStarted = payload => ({ payload, type: FETCH_START });
+export const fetchSuccess = payload => ({ payload, type: FETCH_SUCCESS });
+export const fetchError = payload => ({ payload, type: FETCH_ERROR });
+export const updateStatus = payload => ({ payload, type: UPDATE_STATUS });
+export const updateError = payload => ({ payload, type: UPDATE_ERROR });
+
+/* thunk creators */
+export const fetchFromAPI = () => {
+  return (dispatch, getState) => {
```

Comment:
getState nieużuwane

Reply:
No nieuzywane, to prosto z Kodilli skopiowane było. Usunąłem